### ISSUE TITLE
No email sent to new user when its account is activated from the backend in an admin approval workflow

### ISSUE THREAD

**Issue by Delemo** [joomla/joomla-cms#10216]
Hello,

#### Steps to reproduce the issue
In Joomla! Users options, allow user registration and set New User Account Activation to Administrator

#### Expected result
From the backend, when admin clicks Enabled button for a new user, an email should be sent to this user to notify him that his account is enabled.

#### Actual result
The workflow is working only from the frontend, when clicking the link sent to admin after user registration.

#### System information (as much as possible)
Joomla! 3.5.1
PHP 7.0
MySQL 5.5

Regards

**Comment by photodude** [comment]
What are your Global Settings → Server → Mail Settings?

**Comment by Delemo** [comment]
Hi photodude,

Send Mail: Yes
Mailer: SMTP
Disable Mass Mail: No
SMTP Authentication: Yes
SMTP Security: SSL
And the needed credential info (Port, Username, Password and Host)

Send Test Mail is working fine.

Regards<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10216">issues.joomla.org/joomla-cms/10216</a>.</sub>

**Comment by photodude** [comment]
Do you get this issue with `Mailer: PHP Mailer` ?

**Comment by brianteeman** [comment]
Pretty sure we have had this issue before with email notifications not being sent when a user is created in the admin. Maybe you can search and find it.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10216">issues.joomla.org/joomla-cms/10216</a>.</sub>

**Comment by photodude** [comment]
@brianteeman Is this the issue https://github.com/joomla/joomla-cms/issues/4376 you are referring to?

**Comment by brianteeman** [comment]
I think so - looks familiar

On 3 May 2016 at 19:34, Walt Sorensen <notifications@github.com> wrote:

> @brianteeman <https://github.com/brianteeman> Is this the issue #4376
> <https://github.com/joomla/joomla-cms/issues/4376> you are referring to?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/joomla/joomla-cms/issues/10216#issuecomment-216623138>
>



-- 
Brian Teeman
Co-founder Joomla! and OpenSourceMatters Inc.
http://brian.teeman.net/

**Comment by Delemo** [comment]
@photodude 

I'm working on a MAMP local install and I've never succeed to make PHP Mailer working with MAMP that's why I'm using SMTP configuration.

I will move my test site live and will test PHP Mailer today.

Regards<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10216">issues.joomla.org/joomla-cms/10216</a>.</sub>

**Comment by Delemo** [comment]
Hi,

I've moved my test site on a live web host.
The configuration is the following:
Joomla! 3.5.1
PHP 5.6.20
MySQLi 5.5.46-0+deb7u1-log

I've changed my mail settings for PHP Mail (Send Test Mail passed).

In the backend after clicking Enabled button no email is sent to the user (neither after clicking Activated button).

Using the activation link sent to the admin to activate the user account from the frontend is still working and, by this mean, an email is sent to the user.

Regards<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10216">issues.joomla.org/joomla-cms/10216</a>.</sub>

**Comment by photodude** [comment]
@Delemo Thanks for working through those scenarios of SMTP vs PHPMailer.

**Comment by Devportobello** [comment]
I think this could be done here: (workflow)
https://github.com/joomla/joomla-cms/blob/staging/administrator/components/com_users/models/user.php#L574
https://github.com/joomla/joomla-cms/blob/staging/plugins/user/joomla/joomla.php#L160

**Comment by sovainfo** [comment]
Suggest to close this issue as Works As Designed !

Propose some code if you desire this functionality!

**Comment by joris85** [comment]
I have the same issue, you should have an option in backend to activate and send e-mail.
It is working as designed but if you do not want to do it with e-mail but loggin in in backend and do it there it is just nog possible.

Is there already code for this or is this coming in a coming release?

Thanks!
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10216">issues.joomla.org/tracker/joomla-cms/10216</a>.</sub>

**Comment by brianteeman** [comment]
Unfortunately not - see #13620

For a possible solution read this post and the comments https://brian.teeman.net/joomla/authorising-and-approving-joomla-users

**Comment by Ooops-404** [comment]
@brianteeman 

Is your work around still valid? or is there a different way?
I was going to go through the steps you lay out in your website but then I noticed that you warned that you are not maintaining this. So I prefer to ask before I do anything to the website.


https://brian.teeman.net/joomla/870-authorising-and-approving-joomla-users
https://github.com/brianteeman/AdminApproval/releases/tag/1.0


Sidenote: Not a big deal but I had to search for the article in your website, the URL is now different.
